spec/features/audit/report/allocation_spec.rb

```diff
@@ -1,31 +1,34 @@
 RSpec.feature "Content Allocation", type: :feature do
-  let!(:my_organisation) do
-    create(
-      :organisation,
-    )
-  end
-
-  let!(:me) do
-    create(
-      :user,
-      organisation: my_organisation,
-    )
+  scenario "Filter allocated content" do
+    given_i_am_auditing_a_content_item
+    then_i_see_all_content_items
+    given_that_i_filter_by_attributes_that_do_not_apply_to_any_content_items
+    then_i_see_no_content_items
   end
 
-  scenario "Filter allocated content" do
-    create(
+  def given_i_am_auditing_a_content_item
+    organisation = create(:organisation)
```

Comment:
I find it a bit odd hiding all this setup in this step. For example, this step has nothing to do with me belonging to an organisation (or existing for that matter!). I'd expect this to exist either as some sort of common setup step like`given_i_am_an_auditor_belonging_to_an_organisation`

Reply:
As a corollary to this, you may well need to make `user` and `organisation` global variables, unless we've come up with a smarter way of doing that somewhere else?